test/testing_test.rb

```diff
@@ -166,6 +166,41 @@ def plan_input
       end
     end
 
+    describe "in thread executor hanging test" do
```

Comment:
`describe "in thread executor hanging test"` could be `describe "in thread executor with unrelated events in clock"`